### DISCUSSION TITLE
Show Progress Bar

### DISCUSSION THREAD

**Discussion by jiiins** [blakeblackshear/frigate#15144] [Ask A Question]
Is there a way to display the progress bar when playing a recording, like in the previous versions? Otherwise, I can't tell how long a clip is.

Thanks

**Comment by NickM-27** [comment]
The timeline on the right moves the handle as it plays and the timeline has shading to indicate when the activity is over

**Comment by didi3r** [comment]
Would be nice if the history shows the detection icons on the top-left corner of the video similar to the home page.
